From: Christian Kastner <ckk@debian.org>
Date: Wed, 8 Jul 2026 22:19:35 +0200
Subject: Flush the log before exiting

The reproducible builds effort discovered that on certain arm64 hosts,
messages logged with LOG() did not make it to stdout.
---
 common/arg.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common/arg.cpp b/common/arg.cpp
index 59cfe7e..f51e469 100644
--- a/common/arg.cpp
+++ b/common/arg.cpp
@@ -1274,6 +1274,7 @@ common_params_context common_params_parser_init(common_params & params, llama_ex
             for (size_t i = 0; i < models.size(); i++) {
                 printf("%4zu. %s\n", i + 1, models[i].to_string().c_str());
             }
+            common_log_flush(common_log_main());
             exit(0);
         }
     ));
